docs/src/faq.md

```diff
@@ -129,6 +129,21 @@ It creates a brand new temporary index to populate the data scraped from your
 website. Once the crawl is successfully achieved, this temporary index overwites
 the old index defined in your configuration with the key `index_name`.
 
+## Why my pages is not indexed or can not be retrieved?
+
+We are crawling your website according to your configuration. It might happen
```

Comment:
```suggestion
We are scraping your website according to your configuration. It might happen
```

Reply:
We want to avoid using this wording but you are right. Will not do it

Reply:
I don't understand your comment, but you're more aware of the word :)

Reply:
We do not want to over promote "scraper" since we have a dedicated crawler

Reply:
Please approve if you do not need to change anything please

Reply:
I thought it was the other way round (removing the word "crawler" here)

Reply:
I understood the same as @Haroenv. We don't want to use the word "crawler" anymore.